src/pages/Editor/containers/editContext.js

```diff
@@ -4,8 +4,8 @@ import DomToImage from "dom-to-image";
 export const EditContext = React.createContext();
 
 const EditContextProvider = (props) => {
```

Comment:
The default page should be the blank page

Reply:
cool